integration_test/mint_module/mint_test.yaml

```diff
@@ -0,0 +1,33 @@
+- name: Mint event triggered
+  inputs:
+    - cmd: seid q mint minter --output --json | jq -r ".denom"
```

Comment:
Can we add some comments for some critical steps about what the command is intended to do?